scaffold/github/workflows/PantheonReviewAppsManual.yml

```
@@ -0,0 +1,72 @@
+name: "Pantheon Review Apps"
+
+on:
+  pull_request:
+    types: [ labeled, unlabeled, opened, synchronize, reopened]
```

```suggestion
    types: [ labeled, opened, synchronize, reopened]
```

From justafish
https://github.com/Lullabot/drainpipe/pull/546#discussion_r1608640047